process/incubation.md

```diff
@@ -140,6 +140,7 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
 * Ecosystem
   * Uses SwiftPM
   * Integrated with critical SSWG ecosystem building blocks, e.g., Logging and Metrics APIs, SwiftNIO for IO
+  * Note that you are **not** required to only depend on packages in the incubation process but you should only include dependencies where they are necessary. Package authors are responsible for ensuring the security of dependencies and must be able to fix any issues if dependency authors are unresponsive to security issues.
```

Comment:
This feels like separate or sub-dot for the how to handle dependencies?

Maybe dependencies is a separate section/discussion entirely? And here we'd just say that a package is allowed to depend on non-sswg recommended libraries?

Reply:
Updated!